test/kitchen/test/integration/common/rspec/spec_helper.rb

```diff
@@ -484,14 +481,16 @@ def fetch_python_version(timeout = 15)
     expect(output).to be_truthy
   end
 
-  # it 'runs Python 3 after python_version is set to 3' do
-  #   result = false
-  #   pythonV = fetch_python_version
-  #   if Gem::Version.new('3.0.0') <= Gem::Version.new(pythonV)
-  #     result = true
-  #   end
-  #   expect(result).to be_truthy
-  # end
+  it 'runs Python 3 after python_version is set to 3' do
+    result = false
+    python_version = fetch_python_version
+    p "Python version (should be 2):"
```

Comment:
Is it normal this line is the same as line 514?

Reply:
No (copy-paste error), but this is debug code, so I'll remove it anyway